### PR TITLE
Drop of PHP 7.4 to comply with server master

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

File: workflow-templates/phpunit-mysql.yml [hunk 1]
```diff
@@ -43,7 +43,7 @@ jobs:
 
     strategy:
       matrix:
-        php-versions: ['7.4', '8.0', '8.1']
+        php-versions: ['8.0', '8.1', '8.2']
```
Comment (ChristophWurst) [inline workflow-templates/phpunit-mysql.yml:46]: this doesn't just drop 7.4 but adds 8.2

Comment (nickvergessen) [reply workflow-templates/phpunit-mysql.yml:46]: Right, adjusted PR title

**Comment by come-nc** [comment]
Too soon to add 8.2 in my opinion, the CI on server is not yet passing on 8.2: https://github.com/nextcloud/server/pull/32463

**Comment by nickvergessen** [comment]
Okay, removed the 8.2 for now from phpunit

**Comment by skjnldsv** [comment]
What about using `8.1` as default instead of `8.0` ? :)

**Review by skjnldsv — APPROVED** [review]

**Comment by come-nc** [comment]
> What about using `8.1` as default instead of `8.0` ? :)

For CI it’s better to default  to the oldest one, to catch any breaking by using too modern syntax or feature.

**Comment by nickvergessen** [comment]
I duplicated my own work...
Duplicate of #99

**Comment by skjnldsv** [comment]
> I duplicated my own work... Duplicate of #99

[image
](https://knowyourmeme.com/memes/trust-nobody-not-even-yourself)

**Pull request closed without merging**